package.json

```diff
@@ -27,7 +28,10 @@
   },
   "dependencies": {
     "electron-debug": "^0.5.1",
-    "nconf": "^0.8.2"
+    "nconf": "^0.8.2",
+    "shelljs": "^0.6.0",
+    "title-case": "^1.1.2",
+    "user-home": "^2.0.0"
```

Comment:
yeah, this is why I should have removed the references in the code files, because I had removed it from the dependencies. I think the electron runtime uses a new enough version of node that we don't need to polyfill Promises.